en/docs/integrate/develop/advanced-development/applying-security-to-a-proxy-service.md

```diff
@@ -159,4 +159,4 @@ Create a Soap UI project with the relevant security settings and then send the r
     - Non Repudiation policy needs only Timestamp and Signature. Confidentiality needs all three : Timestamp, Signature and Encryption.
     - For UsernameToken policy, you do not need to provide a Outgoing WS-Security Configuration. Providing the basic auth configuration is enough.
     
-        <img src="{{base_path}}/assets/img/integrate/apply-security/soapui/invoking-username-token.jpg" width="700">
\ No newline at end of file
+        <img src="{{base_path}}/assets/img/integrate/apply-security/soapui/invoking-username-token.jpg" width="700">
```

Comment:
Best to make this image Zoomable and to make the image size a percentage as opposed to specifying the width in pixels.

Reply:
Fixed. Added same fix to all other images in this page.